### ISSUE TITLE
Unfollow issues

### ISSUE THREAD

**Issue by bvttle** [instabot-py/instabot.py#2094]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x] Bug
- [ ] Feature Request
- [ ] Technical Help
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so, have you tried running the code on your local machine?

#### Operating System? (include version)
- [x] macOS
- [ ] Windows
- [ ] Linux (include flavour)
- [ ] Android (include terminal)

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Exact Python Version?
```
Python 3.7.2
```

#### Pip Version?
```
pip 18.1 from /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pip (python 3.7)
```

#### Description of your issue

```
I have the bot set to follow and unfollow 100 people a day. It currently follows, but doesn't unfollow anyone.. I have a database browser and can see whether the accounts are getting unfollowed, none have been unfollowed.

Any idea why?

Could it be related to these options??     
unfollow_recent_feed=False,
    unfollow_not_following=False,
    unfollow_inactive=False,
    unfollow_probably_fake=False,
    unfollow_selebgram=False,

I disabled these because it was unfollowing accounts I had followed before, and don't want that.

Thanks!
```

#### Config

```
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

from instabot_py import InstaBot

bot = InstaBot(
    login="xxxxxx",  # Enter username (lowercase). Do not enter email!
    password="xxxxxxx",
    like_per_day=1000,
    comments_per_day=0,
    tag_list=['nowherediary', 'filmphotographic', 'multipleexposure', 'filmisnotdead', 'istillshootfilm', 'shootmorefilm', 'keepfilmalive', 'onfilm', 'grainisgood', 'filmforever', '35mm', '35mmfilm', 'filmphotography', 'lomochromepurple','lomopurple', 'dubblefilm', 'analogphotography', 'filmshooters', 'staybrokeshootfilm', 'shootfilmstaybroke', 'filmshooters', 'analoguepeople', 'ishootfilm', 'shootfilm', 'revolog', 'believeinfilm', 'imaginarymagnitude', 'analoguephotography', 'filmphoto', 'analoguephotography', 'filmfeed', 'heyfsc', 'canonae1', 'canona1', 'doubleexposure', 'portra800', 'cinestill800t', 'kodakultramax400', 'multipleexposure', 'incamera', 'vaporwaveaesthetic', 'expiredfilm', 'canona1', 'canonae1', 'canonae1program', 'lomo', 'lomolca'],
    tag_blacklist=["nsfw"],
    user_blacklist={},
    max_like_for_one_tag=50,
    follow_per_day=100,
    follow_time=6 * 60 * 60,
    unfollow_per_day=100,
    unlike_per_day=0,
    time_till_unlike=3 * 24 * 60 * 60,  # 3 days
    unfollow_break_min=15,
    unfollow_break_max=30,
    user_max_follow=0,
    unfollow_recent_feed=False,
    unfollow_not_following=False,
    unfollow_inactive=False,
    unfollow_probably_fake=False,
    unfollow_selebgram=False,
    database_name="db3.db",
    # If True, the bot will also unfollow people who dont follow you using the recent feed. Default: True
    # session_file=False, # Set to False to disable persistent session, or specify custom session_file (ie ='myusername.session')
    user_min_follow=0,
    log_mod=0,
    proxy="",
    # List of list of words, each of which will be used to generate comment
    # For example: "This shot feels wow!"
    comment_list=[["so good!", "amazing!", "cool shot", "nice shot", "love this"]
    ],
    # Use unwanted_username_list to block usernames containing a string
    # Will do partial matches; i.e. 'mozart' will block 'legend_mozart'
    # 'free_followers' will be blocked because it contains 'free'
    unwanted_username_list=[
        "asdas"
    ],
    unfollow_whitelist=["example_user_1", "example_user_2"],
    # Enable the following to schedule the bot. Uses 24H
    # end_at_h = 23, # Hour you want the bot to stop
    # end_at_m = 30, # Minute you want the bot stop, in this example 23:30
    # start_at_h = 9, # Hour you want the bot to start
    # start_at_m = 10, # Minute you want the bot to start, in this example 9:10 (am).
)

bot.mainloop()

```

**Comment by dimisdas** [comment]
> Could it be related to these options??     
> unfollow_recent_feed=False,
>     unfollow_not_following=False,
>     unfollow_inactive=False,
>     unfollow_probably_fake=False,
>     unfollow_selebgram=False,

Yes.

> I disabled these because it was unfollowing accounts I had followed before, and don't want that.

You only need `unfollow_recent_feed=False` then. Reenable the rest. Delete the username.db file from your folder as well.

**Comment by gershee** [comment]
I had similar issue.
Users I had manually followed were being unfollowed by the bot.
After reading the solution here (and one other thread), I set 
 unfollow_recent_feed=false
and deleted the username.db file.
upon running the bot again followers that were present before the BOT are STILL being added to the username.db.
attached is a screenshot. this is a clean run, that is i deleted the db and this is the first run.
it's still adding previous people i was following(that were not added by the bot)
![Screenshot_20190513-221325](https://user-images.githubusercontent.com/34194533/57666268-6bde7f80-75cd-11e9-9cd4-eda88de0da7b.jpg)
 to the database!
